packages/block-editor/src/components/preview-options/README.md

```diff
@@ -1,94 +0,0 @@
-# Preview Options
-
-The `PreviewOptions` component displays the list of different preview options available in the editor.
-
-It returns a [`DropdownMenu`](https://github.com/WordPress/gutenberg/tree/HEAD/packages/components/src/dropdown-menu) component with these different options. The options currently available in the editor are Desktop, Mobile, Tablet and "Preview in new tab".
-
-![Preview options dropdown menu](https://make.wordpress.org/core/files/2020/09/preview-options-dropdown-menu.png)
-
-## Table of contents
-
-1. [Development guidelines](#development-guidelines)
-2. [Related components](#related-components)
-
-## Development guidelines
-
-### Usage
-
-Renders the previews options of the editor in a dropdown menu.
-
-```jsx
-import { Icon, MenuGroup } from '@wordpress/components';
-import { PostPreviewButton } from '@wordpress/editor';
-import { __experimentalPreviewOptions as PreviewOptions } from '@wordpress/block-editor';
```

Comment:
Although this component has a `__experimental` prefix, I am concerned that this PR may impact consumers using this component. My understanding is that once an API is exposed to the core, it should remain backward compatible regardless of the prefix, what do you think?

However, [this component does not seem to be used](https://github.com/search?q=org%3Awp-plugins%20__experimentalPreviewOptions&type=code), at least for the plugins published in the plugin directory.

Reply:
Yes, I had already checked that it was not used. And for experimental and stable ones, they were introduced with the intent that they can be changed, so there's an exception where we can assess for each one of these whether it makes sense to keep them or not. I chose to keep it but deprecate it without rendering anything, that way it's not an error that would produce a white page or something serious but a component that won't be rendered. I also don't really expect third-parties to reach for this component to be honest.

Reply:
Oh, I missed that deprecation was [implemented](https://github.com/WordPress/gutenberg/pull/56921/files#diff-61a1980577fb29dd7a7ed4d0a4c23919d5c6bbae93517a9957f80e1c04d50392R6-R11) and it was returning empty. I also don't think there is a case where third-party developers use this component, and as long as it does not return a critical error, I think there is no problem.